python/tvm/relay/op/image/_image.py

```diff
@@ -109,6 +136,23 @@ def compute_affine_grid(attrs, inputs, out_dtype):
 
 reg.register_injective_schedule("image.affine_grid")
 
+@script
+def _affine_grid_func(data, target_shape):
+    out = output_tensor((4,), "int64")
+    out[0] = int64(data[0])
+    out[1] = int64(2)
+    out[2] = int64(target_shape[0])
```

Comment:
do we support different layout for `affine_grid`?

Reply:
No I don't think so.